lib/models/slack-workspace.js

```diff
@@ -12,7 +12,7 @@ const encrypted = EncryptedField(Sequelize, process.env.STORAGE_SECRET);
 module.exports = (sequelize, DataTypes) => {
   const SlackWorkspace = sequelize.define('SlackWorkspace', {
     slackId: {
-      type: DataTypes.BIGINT,
+      type: DataTypes.STRING,
```

Comment:
WHAT HOW DID THIS EVER WORK?!?

Great find 💯 

Note for other viewers that the schema in the db has the right type but for some reason the model here has it wrong. DDL from Postico:

```
-- Table Definition ----------------------------------------------

CREATE TABLE "SlackWorkspaces" (
    id bigint DEFAULT nextval('"SlackWorkspaces_id_seq"'::regclass) PRIMARY KEY,
    "slackId" character varying(255) NOT NULL,
    "createdAt" timestamp with time zone NOT NULL,
    "updatedAt" timestamp with time zone NOT NULL,
    secrets bytea
);

-- Indices -------------------------------------------------------

CREATE UNIQUE INDEX "SlackWorkspaces_pkey" ON "SlackWorkspaces"(id int8_ops);
CREATE INDEX slack_workspaces_slack_id ON "SlackWorkspaces"("slackId" text_ops);
```